lib/docker-toolbelt.coffee

```diff
@@ -1,7 +1,7 @@
 crypto = require 'crypto'
 Promise = require 'bluebird'
 Docker = require 'dockerode'
-semver = require 'semver'
+rSemver = require 'resin-semver'
```

Comment:
I think it's fine to name it just `semver` if we consistently use `resin-semver` everywhere (which is the purpose of this pr and resin-semver).